ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot

```diff
@@ -41,21 +41,21 @@ ${UPGRADE_CONFIG_MAP}    upgrade-config-map
 *** Test Cases ***
 Verify PVC Size
     [Documentation]    Verify PVC Size after the upgrade
-    [Tags]      Upgrade
+    [Tags]      Upgrade    Dashboard
```

Comment:
This is not a UI test

Reply:
Imo same as before, dashboard is the only thing setting this

https://github.com/red-hat-data-services/ods-ci/blob/78b203b5cf97de9c3175b2d64cd0e149de942153/ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot#L42-L47